foruma-frontend/src/stories/Button.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <button type="button" :class="classes" @click="onClick" :style="style">{{ label }}</button>
+</template>
+
+<script>
+import './button.css';
+import { reactive, computed } from 'vue';
+
+export default {
+  name: 'my-button',
+
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    primary: {
+      type: Boolean,
+      default: false,
+    },
+    size: {
+      type: String,
+      validator: function (value) {
+        return ['small', 'medium', 'large'].indexOf(value) !== -1;
```

Comment:
Codacy has a fix for the issue: [Replace `'small',·'medium',·'large'` with `"small",·"medium",·"large"`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
        return ["small", "medium", "large"].indexOf(value) !== -1;
```